docs/installing.adoc

```diff
@@ -0,0 +1,313 @@
+https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/using.adoc[Using ADB documentation] when making this decision.
++
+** VirtualBox installation instructions are available
+https://www.virtualbox.org/manual/ch02.html#startingvboxonlinux[online
+at the VirtualBox website]. Testing has been done with versions 4.3.34
+and 5.0.0.
++
+A summary of the installation is listed below:
++
+....
+$ sudo dnf -y install dkms
+$ curl -O http://download.virtualbox.org/virtualbox/rpm/fedora/virtualbox.repo
+$ sudo mv virtualbox.repo /etc/yum.repos.d/
+$ sudo dnf -y install VirtualBox-4.3
+$ sudo /etc/init.d/vboxdrv setup
+....
++
+While the latest stable release should work, the majority of testing has
+been done with version 4.3.30.
+** Installing libvirt dependencies can be skipped as they are
+automatically installed together with the `vagrant-libvirt` package.
+Testing has been done with libvirt version 1.2.18 and vagrant-libvirt
+version 0.0.32.
+
+* CentOS
++
+Two different virtualization providers are supported on Linux;
+https://www.virtualbox.org[VirtualBox] and http://libvirt.org/[libvirt].
+The choice as to which to use should be driven by your preferences and
+environmental concerns and is outside of the scope of this document.
+Both will work equally well in their default configuration. You may wish
+to read the section onfile synchronization in the
+https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/using.adoc[Using ADB documentation] when making this decision.
++
+** VirtualBox installation instructions are available
+https://www.virtualbox.org/manual/ch02.html#startingvboxonlinux[online
+at the VirtualBox website].
+https://wiki.centos.org/HowTos/Virtualization/VirtualBox[CentOS specific
+instructions] are also available.
++
+While the latest stable release should work, the majority of testing has
+been done with version 4.3.30.
++
+A summary of the installation is listed below:
++
+....
+$ sudo yum -y install epel-release
+$ sudo yum -y install dkms
+$ curl -O http://download.virtualbox.org/virtualbox/rpm/rhel/virtualbox.repo
+$ sudo mv virtualbox.repo /etc/yum.repos.d/
+$ sudo yum -y install VirtualBox-4.3
+$ sudo /etc/init.d/vboxdrv setup
+....
+** Installing libvirt dependencies can be skipped as they are
+automatically installed together with the CentOS software collections
+build.
+
+[[install-vagrant]]
+== Install Vagrant
+
+* Microsoft Windows
+1.  Follow the directions at
+https://docs.vagrantup.com/v2/installation/index.html[vagrantup.com].
+Testing has been done with version 1.7.4.
+2.  Install an `ssh` client. Two good options are:
+** https://cygwin.com/install.html[Cygwin]
+** http://www.mingw.org/[mingw]
++
+Please note, Putty is not recommended as it doesn't currently interface with Vagrant.
+
+* Mac OS X
++
+Follow the directions at
+https://docs.vagrantup.com/v2/installation/index.html[vagrantup.com].
+Testing has been done with version 1.7.4.
+
+* Fedora 22/23/Rawhide(24)
```

Comment:
@budhrg I couldn't cherry-pick the commit via git (don't ask, long story), so I looked into the original PR and manually edited the file to match the version numbers. Just a side note, this is why I asked for everyone to wait with editing content files while I was working, but luckily it wasn't a major change so now it'll be resolved once I update the PR.